### PR TITLE
FD-300: Wallet page - Mobile view - Recent transactions

### PR DESCRIPTION
**Pull request by BigBadAlien**

### CODE REVIEW

File: src/screens/Wallet/WalletSM.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          </ButtonBase>
+        </SwapTokensContainer>
+      </Grid>
+
+      <Grid item={true} xs={12}>
+        <OperationsContainer onRetry={onRetry} />
+      </Grid>
+
+      <Grid item={true} xs={12}>
+        <PriceContainer PanelWithHeaderProps={{ variant: 'solid-shadowed' }} />
```
Comment (starkyru) [inline src/screens/Wallet/WalletSM.tsx]: move this object to const.

File: src/screens/Wallet/WalletSM.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      </Grid>
+
+      <Grid item={true} xs={12}>
+        <TransactionsContainer
+          PanelWithHeaderProps={{ variant: 'solid-shadowed' }}
```
Comment (starkyru) [inline src/screens/Wallet/WalletSM.tsx]: move this object to const.

File: src/screens/Wallet/WalletSM.tsx (hunk 1 above)
Comment (starkyru) [reply src/screens/Wallet/WalletSM.tsx]: actually, I thought a little... What's the reason of sending header props to the PanelContainer? I think there should be a `variant` prop

Comment (starkyru) [reply src/screens/Wallet/WalletSM.tsx]: Parent component shouldn't care about the details of the realisation of the child components.

File: src/screens/Wallet/WalletSM.tsx (hunk 2 above)
Comment (starkyru) [reply src/screens/Wallet/WalletSM.tsx]: same here

File: src/screens/Wallet/panels/Operations/OperationsSM.tsx [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+  ERC20: iconErc20,
+  MAINNET: iconMainnet,
+};
+
+interface IPaymentDetails {
+  payment: IPayment;
+  classes?: Classes;
+}
+
+const PaymentDetails = withStyles(operationsSMStyles)(
```
Comment (starkyru) [inline src/screens/Wallet/panels/Operations/OperationsSM.tsx]: Too many components in this file.

File: src/screens/Wallet/panels/Operations/Operations.tsx [hunk 4]
```diff
@@ -146,7 +154,10 @@ const OperationsComponent: React.FunctionComponent<IOperationsProps> = ({
 
   return (
     <div className={classes.root}>
-      {renderOperationsList()}
+      <Hidden mdUp={true}>
+        {statusElement || <OperationsSM payments={payments || []} />}
```
Comment (starkyru) [inline src/screens/Wallet/panels/Operations/Operations.tsx]: `statusElement || ` can be moved bit higher (there is the same code below)

File: src/screens/Wallet/WalletSM.tsx (hunk 1 above)
Comment (BigBadAlien) [reply src/screens/Wallet/WalletSM.tsx]: Just a material standard way to configure inner standard components

Comment (BigBadAlien) [reply src/screens/Wallet/WalletSM.tsx]: From my point of view, it's just very easy for refactoring. Especially with panels, papers, such things which constantly moving

File: src/screens/Wallet/panels/Operations/Operations.tsx (hunk 4 above)
Comment (starkyru) [reply src/screens/Wallet/panels/Operations/Operations.tsx]: Why normally you use render function and for smaller version - component?

Comment (starkyru) [reply src/screens/Wallet/panels/Operations/Operations.tsx]: if render function has the same interface as the OperationsSM, you could add <Operations> component with the same interface which will decide which version to render.

File: src/screens/Wallet/panels/Operations/OperationsSM.tsx [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+}
+
+const OperationsSMComponent = ({
+  classes = {},
+  payments,
+}: IOperationsSMProps) => {
+  const groupedPayments = groupBy(payments, 'fromAccountAddressType');
+
+  return (
+    <>
```
Comment (starkyru) [inline src/screens/Wallet/panels/Operations/OperationsSM.tsx]: No need for a fragment here

File: src/screens/Wallet/panels/Operations/OperationsSM.tsx [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+const OperationsGroup = withStyles(operationsSMStyles)(
+  ({ classes = {}, token, payments }: IOperationsGroupProps) => {
+    const [opened, setOpened] = useState(false);
+
+    const triggerOpen = () => {
+      setOpened(!opened);
+    };
+
+    return (
+      <>
```
Comment (starkyru) [inline src/screens/Wallet/panels/Operations/OperationsSM.tsx]: No need for a fragment here

**Pull request merged**